core/embed/rust/src/ui/component/text/op.rs

```diff
@@ -238,7 +241,9 @@ impl<T: StringType + Clone> OpTextLayout<T> {
 #[derive(Clone)]
 pub enum Op<T: StringType> {
     /// Render text with current color and font.
-    Text(T),
+    /// Bool signifies whether the text is complete or not
+    /// (so we know whether to show ellipsis or not).
+    Text((T, bool)),
```

Comment:
why not make a separate op for ellipsis instead?

Reply:
How would it then look like? As we cannot know beforehand at which position the ellipsis will be?

To make sure, the use-case of this change is the following:
- 1. we want to show both ellipses (forward and backward one) when we are splitting coherent text (single `Op::Text`, basically one sentence) into more pages

![image](https://github.com/trezor/trezor-firmware/assets/42543243/9b31cd9c-30ba-43c5-a0d7-9c06cb5c7589)

- 2. we do not want to show any ellipsis when one coherent text ends at the end of one page and second `Op::Text` (sentence) starts at the beginning of the second page

![image](https://github.com/trezor/trezor-firmware/assets/42543243/6b78de34-40c4-4566-a810-5f3ce7646a76)

Reply:
i have a _slightly_ cleaner solution in mind, but it seems to require larger modifications to the Op processing pipeline, for which this is not a good place

for now, just modify the `Text` to not use the tuple. Rename `is_complete` to `continued` and update the comments accordingly, to make it clear that this only happens when it's a `Text` item that is continuing from previous page.

Reply:
Done in [3006d28](https://github.com/trezor/trezor-firmware/pull/3107/commits/3006d285ae8e2eba439680480ffd48cc3b176dae)